src/core/plugin/plugin.service.js

```diff
@@ -0,0 +1,22 @@
+import {AppError} from '@grid/core/infrastructure';
+
+export function resolve(model, name) {
```

Comment:
please Make it as a class
class PluginService{
constructor(model);
resolve(name);
}